### ISSUE TITLE
Add sorting in tables, type Option

### ISSUE THREAD

**Issue by artemg2006** [Make-md/makemd#216]
In Notion, for example, it is possible to move up or down buttons when creating a column in tables with the Option type. Now they are fixed in Make and it is impossible to change their sorting to make it convenient. Attached a video of how it works in Notion. It would be great to make such functionality. Thanks!

https://github.com/Make-md/makemd/assets/6279961/22287464-a2aa-44be-b59a-f1dfd9a39e03

**Comment by jp-cen** [comment]
This is committed for an upcoming patch (not sure if you're the same person who suggested it in discord). Will follow up once it's implemented.

**Comment by artemg2006** [comment]
> This is committed for an upcoming patch (not sure if you're the same person who suggested it in discord). Will follow up once it's implemented.

No, it wasn't me, I rarely go to Discord. I searched on Github, but I didn't see that it was already planned. Thank you very much! I'm really looking forward to it!

**Comment by Alcinzal** [comment]
I am not 100% sure if we have the same suggestion here, but I would really love to be able to move rows up and down in a table. Like either by dragging it or right clicking and clicking move up, move down, move to the top or move to the bottom. Because currently when I have big tables and want to add something near the top, I have to use SQLite Browser to edit the context.mdb file for that table, so I dont manually have to edit all the cells.

**Comment by jp-cen** [comment]
You can drag and drop to reorder context tables at the moment by reordering them in the Navigator, both of them use the index as rank. Non context tables currently can’t be sorted. The next release has drag to reorder list view (all the non table views), adding that same functionality to table view is planned.
